res/css/views/settings/tabs/user/_PreferencesUserSettingsTab.scss

```diff
@@ -42,7 +42,7 @@ limitations under the License.
             font-size: $font-15px;
             line-height: $font-18px;
             color: $primary-content;
-            margin: 16px 0;
+            margin: 16px 10px;
```

Comment:
Hmm, why does this PR modify the Community Migrator in a different tab to Labs?

Reply:
> 

Actually, I by mistakenly squashed some commits :sweat_smile:, removing it.

Reply:
> 

Sorry for the multiple force pushes.